src/Photos/Enums.cs

```diff
@@ -378,12 +382,35 @@ public enum FigExifCustomRenderedValue : short {
 	[TV (13,0), Mac (10,15), iOS (13,0)]
 	[Native]
 	public enum PHPhotosError : long {
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'InternalError' instead.")]
 		Invalid = -1,
```

Comment:
No big deal, but you can add a `#if !XAMCORE_4_0` around the Invalid value.

Reply:
ok! adding it in, but curious - why does that help here?

Reply:
Just so that whenever we can break API (and define XAMCORE_4_0), we don't keep a deprecated enum value for which there's an alternative (with no drawbacks whatsoever).